src/ensembl/src/content/app/browser/browser-storage-service.ts

```diff
@@ -75,7 +75,7 @@ export class BrowserStorageService {
     );
   }
 
-  public getSelectedBrowserTab() {
+  public getSelectedBrowserTab(): { [genomeId: string]: TrackType } {
```

Comment:
Could we rename`TrackType` to something like `TrackPanelTab`?

Reply:
I have renamed `TrackType` to be `TrackSet` and `BrowserTabs` to `TrackPanelTabs`. Also renamed the state property `selectedBrowserTab` to `selectedTrackPanelTab`.